lib/worker/src/rpc.rs

```diff
@@ -261,7 +343,7 @@ impl publications::callback::Server for UploadCallback {
         debug!("publication upload {} finished for {}", self.fname, owner);
 
         let n = self.ns.clone();
-        let p = self.pg_pool.clone();
+        let p: sqlx::Pool<sqlx::Postgres> = self.pg_pool.clone();
```

Comment:
maybe we can do `use sqlx::{Pool, Postgres}` above, so we can reference those directly here if that makes it more readable?